tasks/shell.sh

```diff
@@ -250,12 +249,17 @@ random_hexdump () {
 if test "x$TMPDIR" = "x"; then
   tmp="/tmp"
 else
-  tmp=$TMPDIR
+  tmp=${TMPDIR}
+  # TMPDIR has trailing file sep for OSX test box
+  penultimate=$((${#tmp}-1))
+  if test "${tmp:$penultimate:1}" = "/"; then
+    tmp="${tmp:0:$penultimate}"
+  fi
 fi
 
 # Random function since not all shells have $RANDOM
 if exists hexdump; then
-  random_number=random_hexdump
+  random_number=$(random_hexdump)
```

Comment:
This was adding the string "random_hexdump" to the path instead of a random number.